homeassistant/components/tado/sensor.py

```diff
@@ -139,7 +139,10 @@ def extra_state_attributes(self):
     @property
     def unit_of_measurement(self):
         """Return the unit of measurement."""
-        if self.home_variable == "temperature":
+        if (
+            self.home_variable == "temperature"
```

Comment:
I actually prefer:
```py
        if self.home_variable in ["temperature", "outdoor temperature"]:
```

This way, you don't need to repeat the `self.home_variable`.

Reply:
Changed :)